app/components/Header/index.tsx

```diff
@@ -0,0 +1,159 @@
+import * as React from "react";
+import { styled, useTheme } from "@mui/material/styles";
+import Box from "@mui/material/Box";
+import Drawer from "@mui/material/Drawer";
+import MuiAppBar, { AppBarProps as MuiAppBarProps } from "@mui/material/AppBar";
+import Toolbar from "@mui/material/Toolbar";
+import CssBaseline from "@mui/material/CssBaseline";
+import List from "@mui/material/List";
+import Divider from "@mui/material/Divider";
+import IconButton from "@mui/material/IconButton";
+import ChevronLeftIcon from "@mui/icons-material/ChevronLeft";
+import ChevronRightIcon from "@mui/icons-material/ChevronRight";
+import ListItem from "@mui/material/ListItem";
+import ListItemButton from "@mui/material/ListItemButton";
+import ListItemText from "@mui/material/ListItemText";
+import Link from "next/link";
+import { useUser } from "@auth0/nextjs-auth0/client";
+import { auth0UserContext } from "@/types/userContext";
+
+const drawerWidth = 240;
+
+const menuItemsTemplate = [
+  {
+    name: "דף הבית",
+    href: "/",
+  },
+  {
+    name: "התחבר",
+    href: "/login",
+  },
+  {
+    name: "פרופיל",
+    href: "/profile",
+  },
+];
+
+const menuItemsAfterLogin = [
+  {
+    name: "התנתק",
+    href: "/api/auth/logout",
+  },
+];
+
+interface AppBarProps extends MuiAppBarProps {
+  open?: boolean;
+}
+
+const AppBar = styled(MuiAppBar, {
```

Comment:
Were you able to use the `styled` function? If so, then maybe we can align to that and not use the sx prop.